### ISSUE TITLE
Object-and_structural-level_image&label consists of many pixel value categories

### ISSUE THREAD

**Issue by tsrobcvai** [OSUPCVLab/StructureCrackDataset#1]
Thank you for your generous sharing! The following is my request, I’m trying to use this dataset, but I noticed that the pixel value of the label in Object-and_structural-level_image&label is not the same as the pixel-level_image&label(There are only two types of cracks and background), The label file in Object-and_structural-level_image&label consists of many pixel value categories , Because I am a newcomer who is new to image semantic segmentation, I don’t know how to use such labels. Can you generously provide labels similar to the only two types in the pixel-level dataset ? Thank you very much! I wish you a happy life!

**Comment by Bai426** [comment]
Thank you for your interest in our work. 
There are two image sizes in the training data as you mentioned in the email: 768x768 and 256x256 for pixel-level training and object- and structural- level training, respectively. Please noted that each original image and its label should have the same size when segmentation neural networks are used.  You can resize both the original images and labels to get what you want for your training. 

By the way, if you are interested in my work, new training data will be provided in August after our new paper is published.  the image size varies in these training datasets. Thanks!

**Comment by tsrobcvai** [comment]
Thank you for your prompt reply! Looking forward to your new papers and new data sets. I’m sorry. I think there may be some errors in my statement. The following two pictures are using matlab software to open the label of 
![the object level and the structure level](https://user-images.githubusercontent.com/73679440/121370356-40c2e780-c96f-11eb-8090-1192d5208d75.png)
the pixel level dataset and object- and structural-level dataset respectively (This is what I used to see The method of each pixel of the label map). From this, we can see that there are only two types (0 and 255) for the label of the piexl level ![pixel-level](https://user-images.githubusercontent.com/73679440/121370347-3ef92400-c96f-11eb-9dce-587b3813f920.png) data set, but there are many kinds of pixel values for the object level and the structure level. This should It will cause the 2 classification semantic segmentation network to report an error, This is my doubt,  Looking forward to your reply!

**Comment by Bai426** [comment]
Okay, I understand what you are talking about. 
Yes, they both are binary for the pixel-level label and the labels of object and structural level, but the background is in black and purple while the cracks are in white and yellow, respectively. I didn't have any problem with these training data for the U-Net, which seems to turn these labels into gray images at first and the background and foreground (cracks) will be distinguished.  Therefore, you can try them on your networks at first. If it works, you can keep the format of the labels. If not, e.x., the labels for object level and structural level, you can convert them into black and white labels. The last option is easy because it is just like a process to convert these colorful images to gray images, or it is like a binarization process. Please let me know if you have any questions. Thanks!

**Comment by tsrobcvai** [comment]
Ok, I understand, I will try the method you said, thank you！
